query_optimizer/resolver/Resolver.cpp

```diff
@@ -724,6 +708,25 @@ const S::PartitionSchemeHeader* Resolver::resolvePartitionClause(
     THROW_SQL_ERROR_AT(partition_type_string) << "Unrecognized partition type: " << partition_type;
   }
 
+  proto->set_num_partitions(partition_clause->num_partitions()->long_value());
```

Comment:
Do we perform error checks for the following conditions?
1. Number of partitions = 0
2. No attribute specified in the partitioning attributes' list.